### ISSUE TITLE
Voice calls: no sound

### ISSUE THREAD

**Issue by meinhardh** [tieto/sipe#168]
I'm running Ubuntu 17.04 and am trying to establish voice calls through my organisation's Lync/S4B servers. The authentication works fine (I can see my own out-of-office message I put on the Exchange server), and I can call phone numbers - the remote phone rings, but I don't get any audio on my Ubuntu system. Having tried first the packages in the standard repositories, I have now installed those from sipe-collab, but the result is the same.

I've created a debug log according to the instructions on the Wiki - please provide an address I can send it to. The list of packages installed on my system is attached.
[packages.txt](https://github.com/tieto/sipe/files/1586969/packages.txt)

Any help would be much appreciated.

**Comment by escorponox** [comment]
I'm experiencing this too. I can confirm others can hear me but I get no sound from them.

**Comment by joakim-tjernlund** [comment]
Try playing with test calls, see if you can get sound working there.
I have a long standing bug: Using USB headsets and ALSA, I can easily trigger microphone stop
working just as the call connects, you can see the sound/volume bar stops moving when this happens.

**Comment by LeonardoDG2084** [comment]
I'm experiencing this too, just on phone calls the first minute mic stop working and the capture bar freeze,  but i still hear the people talking, after 1 minute my mic works again.

**Comment by joakim-tjernlund** [comment]
Right, but this is the wrong place for this problem(as it has nothing/little to do with SIPE)
Report it to Pidgin instead.

**Comment by LeonardoDG2084** [comment]
@joakim-tjernlund, You are right, isn't SIPE, but wasn't pidgin, the problem was phono-vlc backend, I installed phonon gstreamer and everthing works!
Thanks!

**Comment by AndreyPavlenko** [comment]
> @joakim-tjernlund, You are right, isn't SIPE, but wasn't pidgin, the problem was phono-vlc backend, I installed phonon gstreamer and everthing works!
> Thanks!

Could you please explain how did you resolve that? I've the same issue.
